float32 x_std
float32 y_std
float32 z_std
float32 R_std
float32 P_std
float32 Y_std
---